CHANGELOG.md

```diff
@@ -29,6 +29,11 @@ You can check your current version with the following command:
     ```
 
 For more information, see [UP42 Python package description](https://pypi.org/project/up42-py/).
+## 2.1.1a5
```

Comment:
To be upadted after parent merge. pyproject to be updated as well.

Reply:
Done.